ansible/roles/matterbridge/handlers/main.yml

```diff
@@ -3,4 +3,4 @@
 - name: restart matterbridge
   docker_container:
     name: "{{ matterbridge_name }}"
-    restart: yes
+    restart: true
```

Comment:
According to the [docs](https://docs.ansible.com/ansible/latest/modules/docker_container_module.html#parameter-restart) this should be `yes`. Why change it?

Reply:
I changed this to true as it's more robust according to yamllint:

```
./handlers/main.yml

  6:14      warning  truthy value should be one of [false, true]  (truthy)
```

We can ignore this warning and use `yes` instead of `true` for the truthy values but I will prefer us to comply with the linting, as it may saves from other bugs in the future.